build/pythontool.py

```diff
@@ -1,9 +1,13 @@
 from waflib import Options
-import os, re
+import os, sys, re
 
 def options(opt):
     opt.load('python')
-    opt.add_option('--disable-python', action='store_false', dest='python',
+    if sys.platform == 'sunos5':
+        opt.add_option('--disable-python', action='store_true', 
```

Comment:
Actually I misspoke earlier... let's make this one `--enable-python`.  Otherwise they don't have a way to actually turn Python support on in Solaris do they?